src/api/app/services/bs_request_service/action_forwarder.rb

```diff
@@ -0,0 +1,29 @@
+module BsRequestService
+  class ActionForwarder
+    def initialize(bs_request)
+      @bs_request = bs_request
+      @bs_request_actions = bs_request.bs_request_actions
+    end
+
+    def forwarding_options
+      forwarding_options = []
+      @bs_request_actions.each do |action|
+        forwarding_options.append(Hash[action.id, BsRequestActionService::Forwardable.new(action).possible_targets])
```

Comment:
instead of `each` and `append` you could call `collect` on `@bs_request_actions` and reduce this method to one line